### ISSUE TITLE
Notification for future-dated donation

### ISSUE THREAD

**Issue by wzachtaylor** [iATSPayments/com.iatspayments.civicrm#389]
We regularly have donors that set up recurring donations that start on a future date. The donors do not receive a notification until the first donation processes. Is there a way for them to receive a notification as soon as it is scheduled and another when it processes? Not sure if this is a CiviCRM issue or an iATS issue so I have posted here and on Stack Exchange.

**Comment by adixon** [comment]
Yeah, that would be a worthwhile feature. In general, emails in response to contributions are the domain of civicrm core, not this extension, though there are some places where this extension makes some fixes to how core behaves, so let's start with the stack exchange post and see what other have to offer as solutions. Can you post a link to your SE question?

I'm not sure if future recurring start dates on public forms is yet supported in core, it was originally only supported in back-end admin forms and this extension provided the ability to offer it for public forms.

**Comment by wzachtaylor** [comment]
@adixon Here is my SE question: https://civicrm.stackexchange.com/questions/41896/notification-for-future-dated-contribution
